include/swift/AST/DiagnosticsSema.def

```diff
@@ -2482,7 +2482,7 @@ NOTE(silence_inject_forced_downcast,none,
      "add parentheses around the cast to silence this warning", ())
 
 ERROR(conditional_downcast_foreign,none,
-      "conditional downcast to CoreFoundation type %0 will always succeed",
+      "conditional downcast to CoreFoundation type %0 will always succeed; did you mean to use 'as!' to force downcast?",
```

Comment:
The bug report ([SR-7015](https://bugs.swift.org/browse/SR-7015)) asks for a change to the wording of the message, so that it mentions using `CFGetTypeId` to check for correctness. Suggesting a force downcast doesn't solve the problem, because the code is trying to query the dynamic type of a CF class.

Reply:
@DougGregor thanks for the feedback. I was wondering what is the best fixit in this case while using `CFGetTypeId` method returns `CFTypeID` type which only can be compared with type-specific functions like `CFStringGetTypeID()`. 
```swift
if CFGetTypeID(obj) == CFStringGetTypeID() {
    let _ = obj as? CFString
}
```
I couldn't find any universal method. Can you give some suggestion ?